signer/tests/integration/postgres.rs

```diff
@@ -4934,7 +4934,7 @@ async fn pending_rejected_withdrawal_no_events() {
 
     // Append some blocks to ensure we have expired requests; we expire the
     // requests in the first 5 canonical blocks, while keeping the others valid.
-    for _ in chain_depth..WITHDRAWAL_BLOCKS_EXPIRY + 5 {
+    for _ in chain_depth..WITHDRAWAL_BLOCKS_EXPIRY as u64 + 5 {
```

Comment:
nit: isn't this cast redundant?

Reply:
It probably came in when i was cherry-picking stuff from the other PR (where i made these changes originally).

Reply:
Removed in [f1e0eed](https://github.com/stacks-network/sbtc/pull/1397/commits/f1e0eed82bc30653280131c2d4b48a2fe256ad40)